### ISSUE TITLE
screenshot-basic is not working on Linux.

### ISSUE THREAD

**Issue by ZeProf2Code** [citizenfx/screenshot-basic#11]
Hello, do the support of Linux will be added to screenshot-basic ? Thanks!

**Comment by dariusu** [comment]
Yes this is a big problem and I would like to know that too. Confirmed a preexisting problem that is CRITICAL. And will crash linux server if you use screenshot-basic. I reported it, but it was never fixed.

**Comment by dwiprizzy** [comment]
so we need Windows server for use this scipft ?

**Comment by Nexxed** [comment]
It will work on Linux, just not as stable as it should. As for getting it to *run*, try building it on Windows and then moving the files across to your Linux box - it worked for us :)

**Comment by dwiprizzy** [comment]
witch file i need to move ?

**Comment by geekparadize** [comment]
Can you send me a build version for linux server please
